requirements.txt

```diff
@@ -1,3 +1,5 @@
+mako
 git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
 git+https://github.com/edx/XBlock.git@xblock-0.5.0#egg=XBlock==0.5.0
+django-statici18n==1.8.0
```

Comment:
Let's add `git+https://github.com/edx/i18n-tools.git` as well since it's required by the introduced make targets.